src/DssSpell.t.sol

```diff
@@ -528,81 +508,86 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(vest.fin(8), block.timestamp);
     }
 
-    function testVestMKR() private { // make public to use
+    function testVestMKR() public {
         VestAbstract vest = VestAbstract(addr.addr("MCD_VEST_MKR_TREASURY"));
-        assertEq(vest.ids(), 23);
+        assertEq(vest.ids(), 24);
 
         uint256 prevAllowance = gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY"));
 
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        assertEq(gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY")), prevAllowance + 803 * WAD);
+        assertEq(gov.allowance(pauseProxy, addr.addr("MCD_VEST_MKR_TREASURY")), prevAllowance + 787.70 ether);
 
         assertEq(vest.cap(), 1_100 * WAD / 365 days);
-        assertEq(vest.ids(), 24);
+        assertEq(vest.ids(), 28);
 
-        // Friday,   1 July 2022 00:00:00 UTC
-        uint256 JUL_01_2022 = 1656633600;
-        // Saturday, 1 July 2023 00:00:00 UTC
-        uint256 JUL_01_2023 = 1688169600;
+        uint256 AUG_01_2022 = 1659326400;
+        uint256 AUG_01_2023 = 1690862400;
+        uint256 SEP_28_2022 = 1664337600;
+        uint256 SEP_28_2024 = 1727496000;
 
-        address GRO_WALLET = wallets.addr("GRO_WALLET");
+        address GOV_WALLET1 = 0xbfDD0E744723192f7880493b66501253C34e1241;
+        address GOV_WALLET2 = 0xbb147E995c9f159b93Da683dCa7893d6157199B9;
+        address GOV_WALLET3 = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
+        address SNE_WALLET = wallets.addr("SNE_WALLET");
 
         // -----
-        assertEq(vest.usr(24), GRO_WALLET);
-        assertEq(vest.bgn(24), JUL_01_2022);
-        assertEq(vest.clf(24), JUL_01_2022);
-        assertEq(vest.fin(24), JUL_01_2022 + 365 days);
-        assertEq(vest.fin(24), JUL_01_2023);
-        assertEq(vest.mgr(24), address(0));
-        assertEq(vest.res(24), 1);
-        assertEq(vest.tot(24), 803 * WAD);
-        assertEq(vest.rxd(24), 0);
-
-        uint256 prevBalance = gov.balanceOf(GRO_WALLET);
-
-        // Give admin powers to test contract address and make the vesting unrestricted for testing
-        giveAuth(address(vest), address(this));
-        vest.unrestrict(24);
-
-        // 20220907 exec: Warp 1/2 year since cliff, ensure vest was accumulated
-        hevm.warp(JUL_01_2022 + 365 days / 2);
-        vest.vest(24);
-        assertEq(gov.balanceOf(GRO_WALLET), prevBalance + (803 * WAD / 2));
```

Comment:
this part isn't covered in the current mkr streams

Reply:
This seemed excessive to me to run on each vest tbh.

Reply:
I'd rather this gets added to some more generic check so there isn't so much code duplication. Just because something was added once before doesn't mean we need to include it.

Reply:
yeah, I agree, it should be moved to base tests.